### DISCUSSION TITLE
Directory '/static' not found on bundle created with pyinstaller

### DISCUSSION THREAD

**Discussion by rafinhacarmo** [zauberzeug/nicegui#2278] [Q&A]
### Question

Hi,

I'm trying to create a standalone app using nicegui as the ui, but I'm running into a problem when I use it as a bundle created with pyinstaller on Mac OS Sonoma 14.0

The script below runs perfectly, but if I try to create a standalone app with pyinstaller it throws an error:

`RuntimeError: Directory '/Users/rafaelcarmo/Desktop/teste/dist/teste/_internal/nicegui/static' does not exist
[3727] Failed to execute script 'teste' due to unhandled exception!`

Here's the script:

```py
from nicegui import ui
import sys

sys.stdout = open('logs.txt', 'w')

ui.label('Hello world!')

ui.run(native=True, reload=False)
```

To create the bundle I simply used a `pyinstaller teste.py` command.

Am I doing anything wrong here? Does anyone know a way to fix the issue?

**Comment by retsyo** [comment]
please read https://nicegui.io/documentation/section_configuration_deployment#package_for_installation

**Comment by rafinhacarmo** [comment]
Thanks for your message @retsyo . I have read it and still wasn't able to fix the issue. Did you notice anything specific I might have missed?

**Reply by retsyo** [reply]
no, you have not read the document carefully

you should not use `pyinstaller teste.py`, please edit `build.py` and run `python build.py`
